README.md

```diff
@@ -75,6 +76,10 @@ Check that all C++ and Cuda unit test files end with `_test.cpp` or `_test.cu`.
 
 Check that markdown filenames do not use dashes
 
+### `check-sys-path-append`
+
+Check that no sys.path.append is used for Python code
```

Comment:
```suggestion
Check that no `sys.path.append` is used in Python code
```

Reply:
Add this but in the yaml file. readme is autogenerated